build.gradle

```diff
@@ -63,7 +76,7 @@ task getLicenseAndMake(dependsOn: ['getLicense', 'make']) {
 
 wrapper {
    gradleVersion = '6.2.2'
-   distributionUrl = "${repositoryURL}/plugins-releases/org/gradle/gradle/${gradleVersion}/gradle-${gradleVersion}-bin.zip"
+   distributionUrl = "http://cache.devops.pros.com/services.gradle.org/distributions/gradle-${gradleVersion}-bin.zip"
```

Comment:
It looks like this used to be configurable. I'm not sure if referencing "http://cache.devops.pros.com/services" qualifies this as being open source.

Reply:
I removed that part and I changed it by a variable.

Reply:
I just removed gradle.